### PR TITLE
Remove markdown-parser debug logging

### PR DESCRIPTION
**Pull request by kylef**
As per https://github.com/apiaryio/drafter/pull/725#pullrequestreview-258008645, this probably isn't usful. Tools like lldb/gdb exist which can make debugging experiance better.

### CODE REVIEW

File: ext/snowcrash/ext/markdown-parser/src/MarkdownNode.cc [hunk 1]
```diff
@@ -78,68 +78,3 @@ const MarkdownNodes& MarkdownNode::children() const
 
     return *m_children;
 }
-
-void MarkdownNode::printNode(size_t level) const
```
Comment (tjanc) [inline ext/snowcrash/ext/markdown-parser/src/MarkdownNode.cc]: I would keep the definition, but perhaps in DEBUG only.

File: ext/snowcrash/ext/markdown-parser/src/MarkdownNode.h [hunk 2]
```diff
@@ -91,9 +90,6 @@ namespace mdp
         /** Destructor */
         ~MarkdownNode();
 
-        /** Prints the node to the stdout */
-        void printNode(size_t level = 0) const;
```
Comment (tjanc) [inline ext/snowcrash/ext/markdown-parser/src/MarkdownNode.h]: The symbol can stay, always.

File: ext/snowcrash/ext/markdown-parser/src/MarkdownParser.cc [hunk 3]
```diff
@@ -57,10 +57,6 @@ void MarkdownParser::parse(const ByteBuffer& source, MarkdownNode& ast)
     m_source = NULL;
     m_sourceLength = 0;
     m_listBlockContext = false;
-
-#ifdef DEBUG
-    ast.printNode();
-#endif
```
Comment (tjanc) [inline ext/snowcrash/ext/markdown-parser/src/MarkdownParser.cc]: Exactly, this is the only lines I would actually delete.

**Comment by kylef** [comment]
@tjanc ok, I've updated the PR to just remove `printNode`, but I've used it from my prior PR so any printNode ends up on stderr instead.

**Pull request merged**